bin/orca.php

```diff
@@ -19,35 +15,37 @@
 ini_set('memory_limit', -1);
 set_time_limit(0);
 
-$input = new ArgvInput();
-$env = $input->getParameterOption(['--env', '-e'], $_SERVER['APP_ENV'] ?? 'prod', TRUE);
-
-$kernel = new Kernel($env, FALSE);
-
-// Handle a cache:clear pseudo command. This isn't implemented as a true console
-// command because a stale or corrupted cache would render it unusable--
-// precisely when it is needed.
-if (in_array($input->getFirstArgument(), ['cache:clear', 'cc'])) {
-  $filesystem = new Filesystem();
-  // Delete cache directory.
-  $cache_dir = $kernel->getCacheDir();
-  $filesystem->remove($cache_dir);
-  $filesystem->mkdir($cache_dir);
-  $filesystem->touch("{$cache_dir}/.gitkeep");
-  // Delete coverage report directory.
-  $coverage_report_dir = __DIR__ . '/../var/coverage-report';
-  $filesystem->remove($coverage_report_dir);
-  $filesystem->mkdir($coverage_report_dir);
-  $filesystem->touch("{$coverage_report_dir}/.gitkeep");
-  exit;
-}
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+$container = new ContainerBuilder();
+
+$loader = new YamlFileLoader($container, new FileLocator());
+$loader->load(__DIR__ . '/../config/services.yml');
 
-// The below approach has been discussed here:
-// @see https://github.com/TravisCarden/stackoverflow-a-50356503
-$kernel->boot();
-$container = $kernel->getContainer();
+$container->setParameter('app.project_dir', dirname(__DIR__));
+$container->setParameter('app.fixture_dir', dirname(__DIR__) . '/../orca-build');
+
+$container->compile(TRUE);
+
+/**
+ * Console application class.
+ *
+ * @var \Symfony\Component\Console\Application $application
+ */
```

Comment:
I feel like the comment is unnecessary here..

```suggestion
/** @var \Symfony\Component\Console\Application $application */
```